dashboard/app/views/levels/_unplug.html.haml

```diff
@@ -1,5 +1,5 @@
 - unplugged = @level.type == 'Unplugged' ? @level.name : @game.name
-- video_key = try_t("data.unplugged.#{unplugged}.video")
+- video_key = @level.video_key || try_t("data.unplugged.#{unplugged}.video")
```

Comment:
Yep, that's the plan!  I'd like to create `.level` files for the existing Unplugged levels and move their video definition to the new `.level` file.  At that point we can remove the old way of getting Unplugged videos from the translation file.